### ISSUE TITLE
A11y_mssql for VSCode_Results Tab_Keyboard: In high contrast mode, focus is not visible on "Results: untitled-3" tab

### ISSUE THREAD

**Issue by Roopesh-T** [microsoft/vscode#184052]
“[Check out Accessibility Insights](https://accessibilityinsights.io/)! – Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** #A11yMAS; #A11yTCS; #A11ySev2;#BM-VisualStudioCode-Win32-Nov2021;#Visual Studio Code Client;#Win11;#FTP;#DesktopApp;

## Environment:
Visual Studio Code – Insider
Version: 1.79.0-insider (user Setup)
OS Version: Microsoft Windows 11 Enterprise (22621.1702)

## Pre Requisit:

Change system high contrast to aquatic.

## Repro Steps:

1. Open Visual Studio Code in your system
2. Navigate to "Extensions" and install "SQl Server (mssql)" extension
3. Press F1 and select MS SqL : Connect and connect to any sql server
4. "Write code (SELECT *
5. FROM sys.objects;) in the new document and press F1 and select ""MS SQL : Execute Query"" "
6. Change the color theme to high contrast(Files-->Preferences-->Color Theme-->High Contrast)
7. Tab till "Results: untitled-1" tab and verify user is able to see the focus visibility in high contrast mode or not

## Actual Result:
In high contrast mode, focus is not visible on "Results: untitled-1" tab

## Expected Result:
In high contrast mode, focus should be visible on "Results: untitled-1" tab

## User Impact:
Keyboard users and low vision users will finds difficult while working if they are unable to see the focus visibility on "Results: untitled-1" tab

## Attachments:
[184052_VSC.zip](https://github.com/microsoft/vscode/files/11624634/184052_VSC.zip)

**Comment by fsteffi** [comment]
ref: https://github.com/microsoft/vscode-mssql/issues/17714

**Comment by meganrogge** [comment]
@daviddossett could you pls look into this?

**Comment by isidorn** [comment]
This issue is an extension bug on the SQL extension. The extension authors of SQL should file this bug in VS Code with using precise VS COde language, and providing reprodicible steps using only VS Code components. Thank you

**Comment by Roopesh-T** [comment]
@isidorn logged new bug https://github.com/microsoft/vscode/issues/188127 with VS code steps. please verify.

**Comment by Roopesh-T** [comment]
#closed;
